salt/modules/ansiblegate.py

```diff
@@ -276,3 +277,99 @@ def list(pattern=None):
     :return:
     '''
     return _resolver.get_modules_list(pattern=pattern)
+
+
+def playbooks(playbook, rundir=None, check=False, diff=False, extra_vars=None,
+              flush_cache=False, forks=5, inventory=None, limit=None,
+              list_hosts=False, list_tags=False, list_tasks=False,
+              module_path=None, skip_tags=None, start_at_task=None,
+              syntax_check=False, tags=None, playbook_kwargs=None):
+    '''
+    Run Ansible Playbooks
+
+    :param playbook: Which playbook to run.
+    :param rundir: Directory to run `ansible-playbook` in. (Default: None)
+    :param check: don't make any changes; instead, try to predict some
+                  of the changes that may occur (Default: False)
+    :param diff: when changing (small) files and templates, show the
+                 differences in those files; works great with --check
+                 (default: False)
+    :param extra_vars: set additional variables as key=value or YAML/JSON, if
+                       filename prepend with @, (default: None)
+    :param flush_cache: clear the fact cache for every host in inventory
+                        (default: False)
+    :param forks: specify number of parallel processes to use
+                  (Default: 5)
+    :param inventory: specify inventory host path or comma separated host
+                      list. (Default: None) (Ansible's default is /etc/ansible/hosts)
+    :param limit: further limit selected hosts to an additional pattern (Default: None)
+    :param list_hosts: outputs a list of matching hosts; does not execute anything else
+                       (Default: False)
+    :param list_tags: list all available tags (Default: False)
+    :param list_tasks: list all tasks that would be executed (Default: False)
+    :param module_path: prepend colon-separated path(s) to module library. (Default: None)
+    :param skip_tags: only run plays and tasks whose tags do not match these
+                      values (Default: False)
+    :param start_at_task: start the playbook at the task matching this name (Default: None)
+    :param: syntax_check: perform a syntax check on the playbook, but do not execute it
+                          (Default: False)
+    :param tags: only run plays and tasks tagged with these values (Default: None)
+
+    :return: Playbook return
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt 'ansiblehost'  ansible.playbook playbook=/srv/playbooks/play.yml
+    '''
+    command = ['ansible-playbook', playbook]
+    if check is True:
```

Comment:
What is the reason to explicitly check the type alongside the value? I.e. why `if check:` is not sufficient?